package.json

```diff
@@ -54,5 +54,6 @@
     "overrides": {
       "@types/react": "19.0.0"
     }
-  }
+  },
+  "packageManager": "pnpm@10.6.4"
```

Comment:
Nit: Wow, there's a property for that!
Is it possible to move them just below the engine?
I think it would be hard to read if it's far away.

Reply:
Good idea!!

Reply:
done!
76704ba15ed8

Reply:
FYI:
pnpm can manage both node and pnpm versions
https://pnpm.io/npmrc#use-node-version
https://pnpm.io/npmrc#manage-package-manager-versions